packages/components/src/components/Table/TableCell.tsx

```diff
@@ -1,26 +1,60 @@
-import { spacingsPx, typography } from '@trezor/theme';
 import { ReactNode } from 'react';
-import styled, { css } from 'styled-components';
+import styled from 'styled-components';
 
-export interface TableCellProps {
-    children: ReactNode;
-    $alignRight?: boolean;
-    $isHeader?: boolean;
-}
+import { typography, spacingsPx, Elevation, mapElevationToBackground } from '@trezor/theme';
 
-export const TableCell = styled(({ $isHeader, $alignRight, ...props }: TableCellProps) =>
-    $isHeader ? <th {...props} /> : <td {...props} />,
-)<TableCellProps>`
+import { useTableHeader } from './TableHeader';
+import { UIHorizontalAlignment } from '../../config/types';
+import { useElevation } from '../ElevationContext/ElevationContext';
+
+export type TableCellProps = {
+    children?: ReactNode;
+    colspan?: number;
+    align?: UIHorizontalAlignment;
+};
+
+const mapAlignmentToJustifyContent = (align: UIHorizontalAlignment) => {
+    const map: Record<UIHorizontalAlignment, string> = {
+        left: 'flex-start',
+        center: 'center',
+        right: 'flex-end',
+    };
+
+    return map[align];
+};
+
+const Cell = styled.td<{ $isHeader: boolean; $elevation: Elevation }>`
     ${({ $isHeader }) => ($isHeader ? typography.hint : typography.body)}
     color: ${({ theme, $isHeader }) => ($isHeader ? theme.textSubdued : theme.textDefault)};
+    text-align: left;
+    padding: ${spacingsPx.sm} ${spacingsPx.lg};
+    max-width: 300px;
+
+    &:first-child {
+        position: sticky;
```

Comment:
It does not work for me. I think `overflow: hidden;` has to be removed from `table`tag

Reply:
It works for me, weird. You have chromium based browser?

Reply:
It worked in Firefox but not Blink-based browsers, fixed.